server/internal/projects/setlogo_test.go

```diff
@@ -0,0 +1,285 @@
+package projects_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/speakeasy-api/gram/server/gen/projects"
+	"github.com/speakeasy-api/gram/server/internal/contextvalues"
+	"github.com/speakeasy-api/gram/server/internal/oops"
+	projectsRepo "github.com/speakeasy-api/gram/server/internal/projects/repo"
+)
+
+func TestSetLogo(t *testing.T) {
```

Comment:
the sub tests are not necessary here. there is nothing shared between them. please convert each to a top level test case.